src/awfy.js

```diff
@@ -1865,41 +571,81 @@ const DEFAULT_SUITES = [
   'speedometer',
   'stylebench',
   'sunspider',
-  'tp6-amazon',
-  'tp6-apple',
-  'tp6-binast-instagram',
-  'tp6-bing',
-  'tp6-docs',
-  'tp6-ebay',
-  'tp6-facebook',
-  'tp6-google',
-  'tp6-google-mail',
-  'tp6-imdb',
-  'tp6-imgur',
-  'tp6-instagram',
-  'tp6-linkedin',
-  'tp6-netflix',
-  'tp6-microsoft',
-  'tp6-office',
-  'tp6-outlook',
-  'tp6-paypal',
-  'tp6-pinterest',
-  'tp6-reddit',
-  'tp6-sheets',
-  'tp6-slides',
-  'tp6-tumblr',
-  'tp6-twitch',
-  'tp6-twitter',
-  'tp6-fandom',
-  'tp6-wikipedia',
-  'tp6-yahoo-mail',
-  'tp6-yahoo-news',
-  'tp6-yandex',
-  'tp6-youtube',
   'webaudio',
   'wasm-godot',
 ];
 
+Object.keys(DESKTOP_SITES).forEach((key) => {
+  const suite = `tp6-${key}`;
+  BENCHMARKS[`tp6-${key}`] = { compare: {}, label: `Tp6 ${DESKTOP_SITES[key]}` };
+  ['Firefox', 'Chromium', 'Chrome'].forEach((app) => {
+    BENCHMARKS[suite].compare[app.toLowerCase()] = {
+      color: COLORS[app.toLowerCase()],
+      label: app,
+      frameworkId: RAPTOR_FRAMEWORK_ID,
+      suite: `raptor-${suite}-${app.toLowerCase()}`,
+      option: 'opt',
+    };
+  });
+  DEFAULT_SUITES.push(suite);
```

Comment:
The is a small issue with the benchmarks sorting for the mobile suites. It sorts by the key, and `Speedometer` ends up in the middle of the tp6 graphs. I'm attaching a printscreen with the potential issue:

![awfy-refactoring-dropdown-not-ordered](https://user-images.githubusercontent.com/51707553/86566359-31bf0280-bf72-11ea-8c24-e3e1da379143.PNG)

This happens because the options are sorted when added to the Picker dropdowns:

```
<Picker
      key="Benchmark selection"
      identifier="benchmark"
      topLabel="Benchmark"
      onSelection={onChange}
      selectedValue={benchmark}
      options={
        CONFIG.views[platform].benchmarks.sort().reduce((res, benchmarkKey) => {
          res.push({
            value: benchmarkKey,
            label: BENCHMARKS[benchmarkKey].label,
          });
          return res;
        }, [{ value: 'overview', label: 'Overview' }])
      }
/>
```

To have more control on the positions I suggest removing benchmark sorting from above, but this might not be wanted on other platforms

Reply:
> There is a small issue with the benchmarks sorting for the mobile suites. It sorts by the key, and `Speedometer` ends up in the middle of the tp6 graphs.

Good find! I've pushed a fix that reintroduces 'tp6m' as a prefix for the mobile pageload benchmarks, which means they're grouped together again. A proper fix would probably be to sort based on the labels instead of the keys, which would also make the order in the page and dropdown match.